h/test/tweens_test.py

```diff
@@ -48,6 +48,7 @@ def test_tween_csp_uri():
     request = DummyRequest()
     request.registry.settings.update({
         'csp.enabled': True,
+        'csp.report_only': False,
```

Comment:
The values in `request.registry.settings` are set only once per session, so these tests were order dependent. Setting `csp.report_only` explicitly removes the order-dependence.